docs/OpenShift.md

```diff
@@ -1,19 +1,29 @@
 # Red Hat OpenShift Configuration
 
 The Splunk Operator will always start Splunk Enterprise containers using
-a specific, unprivileged `splunk` user and group. Running as a specific user
-and group is required for the Splunk Enterprise containers to write to
-Kubernetes PersistentVolumes. This follows best security practices, which
-helps prevent any malicious actor from escalating access outside of the
+a specific, unprivileged `splunk(41812)` user and group to allow write access  to Kubernetes PersistentVolumes. This follows best security practices, 
+which helps prevent any malicious actor from escalating access outside of the
 container and compromising the host. For more information, please see the
 Splunk Enterprise container's
 [Documentation on Security](https://github.com/splunk/docker-splunk/blob/develop/docs/SECURITY.md).
+The Splunk Enterprise pods are attached to the `default` serviceaccount or the configured
```

Comment:
Can this be part of the next paragraph, along with Operator pod's service account explanation?